src/current/v22.2/alter-database.md

```diff
@@ -367,7 +367,9 @@ In CockroachDB, the following are aliases for `ALTER DATABASE ... RESET {session
 
 ### `SET {session variable}`
 
-`ALTER DATABASE ... SET {session variable}` modifies a [session variable](set-vars.html#supported-variables) for a database. By default, session variable values are set for the duration of the current session.
+`ALTER DATABASE ... SET {session variable}` sets the default value of a [session variable](set-vars.html#supported-variables) for all future sessions on the database. This command does **not** alter the session setting of the current session.
+
+By default, session variable values are set for the duration of the current session.
```

Comment:
nit: This sentence confuses me. What does it have to do with `ALTER DATABASE ... SET ...`?

Reply:
Ha, it confused me too and I hedged on removing it until now.